bridge/github/config.go

```diff
@@ -205,15 +206,67 @@ func promptUsername() (string, error) {
 	}
 }
 
-func promptURL() (string, string, error) {
+func promptURL(repo repository.RepoCommon) (string, string, error) {
 	for {
-		fmt.Print("Github project URL: ")
 
-		line, err := bufio.NewReader(os.Stdin).ReadString('\n')
+		fmt.Println("Detected Projects:")
+
+		repos, err := repo.ListRemotes()
+
 		if err != nil {
 			return "", "", err
 		}
 
+		reposMap := make(map[int]string)
+		i := 1
+		for _, value := range repos {
+
+			if checkGithubURL(value) {
+				shortenedURL, err := shortenedURL(value)
+				if err != nil {
+					fmt.Println(err)
+					continue
+				}
+				fmt.Printf("[%d]: %s\n", i, shortenedURL)
+				reposMap[i] = value
+				i++
+			}
+		}
+
+		if len(reposMap) == 0 {
+			fmt.Printf("No github projects found\n")
+		}
+
+		fmt.Printf("\n[0]: Another project\n")
+		fmt.Println()
+		var line string
+		output, err := bufio.NewReader(os.Stdin).ReadString('\n')
+		if err != nil {
+			fmt.Println(err)
+			return "", "", fmt.Errorf("input error")
+		}
+		output = strings.TrimRight(output, "\n")
+
+		outputInteger, _ := strconv.Atoi(output)
+
+		if outputInteger == 0 {
+			fmt.Println("Enter Project URL:")
+			line, err = bufio.NewReader(os.Stdin).ReadString('\n')
+			line = strings.TrimRight(line, "\n")
+
+			if !checkGithubURL(line) {
+				fmt.Printf("Please Enter valid github URL\n")
+				fmt.Println()
+				continue
```

Comment:
since this is printing "Please enter a valid github URL" can we just directly recall the url reader ?

Reply:
We could, but it would be best if we start he procedure all over again.The previous implementation had it this way so IMO this should do just well